pipeline/recon/helpers.py

```diff
@@ -19,6 +21,15 @@ def get_tool_state() -> typing.Union[dict, None]:
         return pickle.loads(tools.read_bytes())
 
 
+def meets_requirements(requirements):
```

Comment:
while i like the idea of staying DRY with a helper function, this solution breaks the tab-completion checking of installed commands (not verified, but pretty darn confident).

Reply:
Tab completion appears to still be functional for all contexts